illink.targets

```diff
@@ -25,6 +25,12 @@
     <ILLinkRewritePDBs Condition="'$(ILLinkRewritePDBs)' == '' AND '$(DebugSymbols)' != 'false'">true</ILLinkRewritePDBs>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(ILLinkTrimXml)' != ''">
+    <EmbeddedResource Include="$(ILLinkTrimXml)">
```

Comment:
/cc @weshaggard 
Do we want to have some convention for this instead?  It seems odd that a project needs to both add an XML file and set a property pointing at that XML file, but you don't want the project to create the EmbeddedResource item.  If it were me, I'd go all one way or the other (so long as you can come up with a good convention).  Potentially have a convention then if I project needs to fork (eg: one xml for netcoreapp, but a different for UAP) then they can set the property.

Reply:
Disregard.  I see we actually already do this.  In fact I set it up that way in https://github.com/erozenfeld/corefx/commit/c42fd5602476dc88fd0d677a23ed15e8592a91e9.  Sorry about that, for some reason I thought this was new property.